M2/cmake/check-libraries.cmake

```diff
@@ -142,7 +142,7 @@ find_package(EAntic	2.0.0 QUIET) # only needed when libnormaliz.so needs it
 find_package(MSolve	0.7.0)
 find_package(Frobby	0.9.0)
 find_package(CDDLIB)  # 0.94m?
-find_package(GTest	1.10)
+find_package(GTest	1.16)
```

Comment:
Why such a high minimum version?  1.16 was only released this February.

Reply:
Because it's the only one compatible with the most recent version of cmake, and we always download gtest anyway, so might as well be the latest one.

Reply:
`find_package` also looks in `usr-host` where it's installed in.

Reply:
Ah, ok.  That makes sense.